docs/sources/writing-guide/shortcodes/index.md

```diff
@@ -143,6 +143,77 @@ This shortcode inserts a lists of links to the page's subpages and includes the
+Any link from alerting documentation to dashboard documentation, which is also published to both Grafana and Grafana Cloud, needs to be specific as to which destination is appropriate for the product documentation.
+Grafana Cloud alerting documentation should link to Grafana Cloud dashboard documentation, and Grafana alerting documentation should link to Grafana dashboard documentation.
+
+The destination used in a rendered page derives from matching path prefixes:
+
+```markdown
+{{%/* docs/reference */%}}
+[<LABEL>]: "<PROJECT PATH PREFIX> -> <REFERENCE>"
+...
+{{%/* /docs/reference */%}}
+```
+
+- _`LABEL`_ matches the link label used in a reference-style link (`[LINK TEXT][LABEL])`).
+  For more information about reference-style links, refer to [Daring Fireball: Markdown Syntax Documentation](https://daringfireball.net/projects/markdown/syntax#link).
+- _`PROJECT PATH PREFIX`_ is matched against the page’s URL path, where the first matching `PROJECT PATH PREFIX` is chosen.
+  The project of the destinations where you want _`REFERENCE`_ to be used.
+  For example, for Grafana, use `/docs/grafana/`, for Mimir, use `/docs/mimir/`.
+- _`REFERENCE`_ is the same parameter used in a `relref` shortcode.
+  It can include the syntax `<SOMETHING VERSION>`, which is first looked up in the front matter before falling back to the version inferred from the page's version.
+  Any of the kinds of arguments documented in [Links and Cross References](https://gohugo.io/content-management/cross-references/#use-of-ref-and-relref) can be used.
+
+### Examples
+
+#### Version interpolation
+
+Given a page in versioned Grafana documentation containing the following the reference-style link:
+
+```markdown
+For more information about Grafana dashboards, refer to [Dashboards][dashboards].
+```
+
+Version interpolation of `/docs/reference` enables the use of absolute paths that resolve correctly, irrespective of version.
+In the following example, the `/docs/reference` shortcode replaces the variable `<GRAFANA VERSION>` with the version inferred from the current page.
+
+- If the page is `/docs/grafana/latest/alerting`, the inferred version is `latest`, and the returned reference is `/docs/grafana/latest/dashboards`.
+- If the page is `/docs/grafana/next/alerting`, the inferred version is `next`, and the returned reference is `/docs/grafana/next/dashboards`.
+
+```markdown
+{{%/* docs/reference */%}}
+[dashboards]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/dashboards"
+{{%/* /docs/reference */%}}
+```
+
+You can override version inference using front matter.
+To override the value of `<GRAFANA VERSION>`, set the `grafana_version` parameter in the page's front matter.
+For example, with the front matter `grafana_version: next`, the shortcode replaces `<GRAFANA VERSION>` with `next`.
+
+#### Contextual destinations
+
+Given a page in versioned Grafana documentation containing the following the reference-style link:
```

Comment:
I'm not following the flow of this section well. The phrase "Given a page..." seems to need a "...then this is true" conclusion, not a colon.